### ISSUE TITLE
`tkctl get tikv` should show store-id and store state

### ISSUE THREAD

**Issue by aylei** [pingcap/tidb-operator#564]
## Feature Request

**Is your feature request related to a problem? Please describe:**

Currently, `tkctl get tikv` don't print store-id and store state of each tikv instance:
```
$ tkctl get tikv
NAME                        READY   STATUS    MEMORY    CPU   RESTARTS   AGE     NODE
stability-cluster2-tikv-0   1/1     Running   2Gi/8Gi   1/8   0          6m43s   172.16.4.154
stability-cluster2-tikv-1   1/1     Running   2Gi/8Gi   1/8   0          7m8s    172.16.4.149
stability-cluster2-tikv-2   1/1     Running   2Gi/8Gi   1/8   0          7m47s   172.16.4.150
```
These two fields are important for inspecting the cluster and diagnosing problems.

**Describe the feature you'd like:**

`tkctl get tikv` print the store-id and store state of each instance.

**Comment by Yisaer** [comment]
It seems difficult to show different columns for each component by `tkctl get tidb / pd /tikv ` as they all have the same print handler. Do you think it's a good idea to put all the extra info one column named `EXTRA_INFO'
here is one example
```bash
$ tkctl get tikv
NAME                        READY   STATUS    MEMORY    CPU   RESTARTS   AGE     NODE         EXTRA_INFO  
stability-cluster2-tikv-0   1/1     Running   2Gi/8Gi   1/8   0          6m43s   172.16.4.154    store-id=1
stability-cluster2-tikv-1   1/1     Running   2Gi/8Gi   1/8   0          7m8s    172.16.4.149    store-id=2
stability-cluster2-tikv-2   1/1     Running   2Gi/8Gi   1/8   0          7m47s   172.16.4.150    store-id=3
```

**Comment by aylei** [comment]
@Yisaer I prefer dedicated columns for information specific to components if possible. The printer maintains a type -> handler mapping and select the handler by the resource type. 

I thought we can use type alias like `type TiKV v1.Pod`, `type TiDB v1.Pod` and register handlers for the type alias, these handlers can share much of code in the `PrintPod` function. However, I have not verified this idea🤣

**Comment by Yisaer** [comment]
Registering different handler for each component type is also my first idea to show dedicated columns. The biggest concern for this idea is it might be a bit more complex. I will try this way to print dedicated columns for each component,meanwhile, make the balance between maintainability and complexity.

**Comment by Yisaer** [comment]
I find it is easy to get each tikv store-id in theri labels but have no idea how to get their own store status. Do you have any idea? @aylei

**Comment by aylei** [comment]
@Yisaer You can get it from the status of the `TidbCluster` obejct

**Comment by Yisaer** [comment]
Thanks. Considering the diffierence between the way to get store-id and store-status for tikv component, I will divide it into two PR. One for store-id ,another for store-status.

**Comment by aylei** [comment]
closed via #916 & #842
